pkg/ingester/wal.go

```diff
@@ -121,6 +152,14 @@ func newWAL(cfg WALConfig, userStatesFunc func() map[string]*userState, register
 		Help:       "Time taken to create a checkpoint.",
 		Objectives: map[float64]float64{0.5: 0.05, 0.9: 0.01, 0.99: 0.001},
 	})
+	w.walLoggedBytes = promauto.With(registerer).NewCounter(prometheus.CounterOpts{
+		Name: "cortex_ingester_wal_logged_bytes_total",
```

Comment:
Name doesn't suggest any failure, like Help does. Same for next metric. In fact, it's not used for failures.

Reply:
Bad copy-paste! That said, I made a separate PR for that https://github.com/cortexproject/cortex/pull/2497 as I was not sure when would this be merged.